src/components/explorer/IndividualExperiments.js

```diff
@@ -89,139 +89,120 @@ const IndividualExperiments = ({ individual }) => {
               </Descriptions.Item>
-              <Descriptions.Item label="Library Layout">{e.library_layout}</Descriptions.Item>
-              <Descriptions.Item label="Library Selection">{e.library_selection}</Descriptions.Item>
-              <Descriptions.Item label="Library Source">{e.library_source}</Descriptions.Item>
-              <Descriptions.Item label="Library Strategy">{e.library_strategy}</Descriptions.Item>
             </Descriptions>
-          </Descriptions.Item>
-          <Descriptions.Item>
-            <Descriptions
-              title="Extra Properties"
-              layout="horizontal"
-              bordered={true}
-              column={1}
-              size="small"
-            >
+            <Descriptions layout="vertical" bordered={true} column={1} size="small">
+              <Descriptions.Item>
+                <Descriptions layout="horizontal" bordered={true} column={1} size="small">
+                  <Descriptions.Item label="Experiment Type">{e.experiment_type}</Descriptions.Item>
+                  <Descriptions.Item label="Study Type">{e.study_type}</Descriptions.Item>
+                  <Descriptions.Item label="Extraction Protocol">{e.extraction_protocol}</Descriptions.Item>
+                  <Descriptions.Item label="Library Layout">{e.library_layout}</Descriptions.Item>
+                  <Descriptions.Item label="Library Selection">{e.library_selection}</Descriptions.Item>
+                  <Descriptions.Item label="Library Source">{e.library_source}</Descriptions.Item>
+                  <Descriptions.Item label="Library Strategy">{e.library_strategy}</Descriptions.Item>
+                </Descriptions>
+              </Descriptions.Item>
               <Descriptions.Item>
-                <JsonView inputJson={e.extra_properties} />
+                <Descriptions
+                  title="Extra Properties"
+                  layout="horizontal"
+                  bordered={true}
+                  column={1}
+                  size="small"
+                >
+                  <Descriptions.Item>
+                    <JsonView inputJson={e.extra_properties} />
+                  </Descriptions.Item>
+                </Descriptions>
               </Descriptions.Item>
             </Descriptions>
-          </Descriptions.Item>
-        </Descriptions>
-    </div>
-    <div className="experiment-titles">
-    <Typography.Text style={titleStyle} level={4}> { `${e.experiment_type} - Results` }</Typography.Text>
-    </div>
-    <Table
-      bordered
-      size="small"
-      pagination={false}
-      columns={EXPERIMENT_RESULTS_COLUMNS}
-      rowKey="filename"
-      dataSource={e.experiment_results.sort((r1, r2) => (r1.file_format > r2.file_format ) ? 1 : -1)}
-    />
-    {i !== (experimentsData.length - 1) && <Divider/>}
-  </div>
-))}
+          </div>
+          <div className="experiment-titles">
+            <Typography.Text style={titleStyle} level={4}>
+              {" "}
```

Comment:
Out if curiosity, could you clarify what this is for? (I believe it adds a regular space?)

Reply:
ah, this is a linter artifact which was already present in the code. Removed!